docs-source/content/samples/simple/domains/domain-home-in-image/_index.md

```diff
@@ -140,14 +140,15 @@ The following parameters can be provided in the inputs file.
 | `domainUID` | Unique ID that will be used to identify this particular domain. Used as the name of the generated WebLogic domain as well as the name of the Kubernetes domain resource. This ID must be unique across all domains in a Kubernetes cluster. This ID cannot contain any character that is not valid in a Kubernetes service name. | `domain1` |
 | `exposeAdminNodePort` | Boolean indicating if the Administration Server is exposed outside of the Kubernetes cluster. | `false` |
 | `exposeAdminT3Channel` | Boolean indicating if the T3 administrative channel is exposed outside the Kubernetes cluster. | `false` |
+| `httpAccessLogInLogHome` | Boolean indicating if server HTTP access log file should be written to the same directory as `logHome` if `logHomeOnPV` is true. Otherwise, server HTTP access log file will be written to directory specified in WebLogic domain home configuration. | `true` |
```

Comment:
Change file to 'files', or add an 'a' before each HTTP.  Dealers choice.

Reply:
For example:  ` Boolean indicating if server HTTP access log files should be written to the same directory as `logHome` if `logHomeOnPV` is true. Otherwise, server HTTP access log files will be written to directory specified in WebLogic domain home configuration. `

Reply:
Actually, the preferred choice is to use plurals, so log file -> log files.
written to directory -> written to the directory
specified in WebLogic domain home -> specified in the WebLogic domain home